recipes-wlroots/wayvnc/aml_0.3.0.bb

```diff
@@ -0,0 +1,16 @@
+SUMMARY = "Andri's Main Loop"
```

Comment:
[meta-toradex](https://github.com/toradex/meta-toradex-tezi/blob/kirkstone-6.x.y/recipes-core/aml/aml_git.bb) have a recipe for `aml`. It adds a `aml-examples` package to add examples if needed.

Reply:
Ok, so maybe it is not a good idea to add it here. Or would you like me to drop aml and neatvnc
but still add wayvnc?

Reply:
You have to add it here, that was just a hint if you want to complete a bit your version of this recipe.